Documentation/gettingstarted/encryption.rst

```diff
@@ -167,6 +169,28 @@ exec -ti cilium-7cpsm -- bash`` and execute the following commands:
     15:16:21.627699 IP 10.60.1.1 > 10.60.0.1: ESP(spi=0x00000001,seq=0x57e4), length 100
     15:16:21.628408 IP 10.60.1.1 > 10.60.0.1: ESP(spi=0x00000001,seq=0x57e5), length 100
 
+Key Rotation
+============
+
+To replace cilium-ipsec-keys secret with a new keys,
+
+.. code-block:: shell-session
+
+    KEYID=$(kubectl get secret -n cilium cilium-ipsec-keys -o yaml|grep keys: | awk '{print $2}' | base64 -d | awk '{print $1}')
+    if [[ $KEYID -gt 15 ]]; then KEYID=0; fi
+    data=$(echo "{\"stringData\":{\"keys\":\"$((($KEYID+1))) "rfc4106\(gcm\(aes\)\)" $(echo $(dd if=/dev/urandom count=20 bs=1 2> /dev/null| xxd -p -c 64)) 128\"}}")
+    kubectl patch secret -n cilium cilium-ipsec-keys -p="${data}" -v=1
```

Comment:
Does this work as is? Don't you need the `op`, `path` of json patch?

Reply:
It worked when I tested it and verified by calling 'get secret' and checking cilium_ipcache map so seems sufficient.